slices/libc6.yaml

```diff
@@ -3,28 +3,28 @@ package: libc6
 slices:
   config:
     contents:
-      /etc/ld.so.conf.d/x86_64-linux-gnu.conf:
+      /etc/ld.so.conf.d/*-linux-*.conf:
```

Comment:
It's a bit unfortunate that x86_64 is not an actual architecture as far as dpkg is concerned. It would be a bit nicer to have this as ${ARCH} or similar. Maybe we can still do this in a similar way.